<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>set Axes</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>601</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="6" rowspan="4">
     <widget class="QComboBox" name="y_d"/>
    </item>
    <item row="1" column="8" rowspan="4">
     <widget class="QLineEdit" name="y_dExp"/>
    </item>
    <item row="1" column="5" rowspan="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>y axis</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="4" colspan="2">
     <widget class="QLineEdit" name="y_nExp"/>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="x_d"/>
    </item>
    <item row="0" column="8">
     <widget class="QLineEdit" name="x_dExp"/>
    </item>
    <item row="1" column="1" rowspan="4">
     <widget class="QComboBox" name="y_n"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="x_n"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLineEdit" name="x_nExp"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>x axis</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" rowspan="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="4">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
